specification/dimensions/region.md

```diff
@@ -0,0 +1,30 @@
+# Region
+
+A Region is an isolated geographic area where a resource is provisioned in and/or a service is provided from. Region is commonly used for scenarios like analyzing cost and prices based on where resources are deployed.
+
+The Region column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
+
+## Column ID
+
+Region
+
+## Display name
+
+Region
+
+## Description
+
+A Region is an isolated geographic area where a resource is provisioned in and/or a service is provided from.
```

Comment:
nit: Do we want to remove the name from the description?
```suggestion
Isolated geographic area where a resource is provisioned in and/or a service is provided from.
```

Reply:
It looks like this change was performed on line 3 instead of line 17 (within Description)